docs/source/project_documentation/aac_gen_gherkin.md

```diff
@@ -0,0 +1,102 @@
+
+```yaml
+model:
+    name: example_name
+    description: Description of Model
+    behavior:
+        - name: example_behavior
+          description: Does something
+          acceptance:
+            - name: example_acceptance
+              scenarios:
+                - name: example scenario
+                  given:
+                    - Initial state
+                  when:
+                    - Something happens
+                  then:
+                    - The model does something in response.
+```
+
+### Gherkin Syntax
+When output to a feature file, a `model`s `behavior` becomes a `feature`, and its `acceptance` becomes a `scenario`.
+
+Each `scenario` has a number of steps:
+
+`given`: An optional step, but is usually recommended to include.  The `given` step describes the models initial state.
+
+`when`: This step describes what kicks off the action the `model` will perform, be it called through a command line or triggered some other way.
+
+`then`: This step describes what happens after the `model` performs its action.  Typically it will include whatever output is created by the `model`'s action.
+
+Along with the `scenario` steps, `features` and `scenario`s can also have requirements, signified by the `requirement ID` before the `feature` or `scenario`, such as the following excerpt taken from the `Math Message Handler` feature file:
+```text
+@SQRT-1
+Feature:
+
+    @SQRT-2
+    Scenario: Request handler issues error for square root command with negative value
+
+        Given The MathMessageHandler is running
+        When The user makes a square root request with MathRequest
+        Then The user gets a MathResponse response with the error
+```
+
+## Plugin Usage Example
+The calculator model system contains three model definitions: `calculator`, `mathlogger`, and `mathmessagehandler`.
+`mathlogger` and `mathmessagehandler` each contain one behavior definition, while calculator contains two.
+The `example_behavior` behavior in the calculator model exists only to demonstrate the gen-gherkin plugins ability to generate multiple feature files per model.
+
+Running the following command will generate a gherkin feature file for each behavior definition in the system.
+
+```bash
+$ aac gen-gherkin-behaviors ./tests/calc/model/calculator.yaml ./output_directory
+```
+It will also return the following output to the command line:
+
+```bash
+All AaC constraint checks were successful.
+Successfully generated feature file(s) to directory: ./output_directory
+```
+If the architecture file provided to the command is not a model, or does not contain a behavior field, it will return the following failure message to the command line:
+
+```bash
+No applicable behavior to generate a feature file
+```
+
+After running the above command, the following four feature files will be generated in `./output_directory`.  The names of the feature files are concatenations of the model name (`calculator`) and the behavior name (`example_behavior`):
```

Comment:
And for this portion, remember to get a screen capture of the updated file tree that shows the new directory and feature files. I know it's spoon feeding, but a chunk of users are not super comfortable with command line, so they need to have what all can they expect to see provided for them.